src/app/+connections/component.service.ts

```diff
@@ -18,11 +17,7 @@ export class ComponentService {
 
   errorMessage: string;
   private allComponents: IComponent[];
-
-  //baseUrl: string;
-  //private componentsUrl = 'app/+components/component.data.json'; // URL to JSON file
-  //private componentsUrl = 'http://localhost:9090';
-  private baseUrl = 'http://localhost:9090/v1';
+  private baseUrl = Globals.apiEndpoint;
```

Comment:
This should come from injected appState like https://github.com/redhat-ipaas/ipaas-client/pull/97/files#diff-5a78858d0f5523c0c33c96923d35ac24R25